joebvp/utils.py

```diff
@@ -1,14 +1,16 @@
+from __future__ import print_function, absolute_import, division, unicode_literals
+
 import numpy as np
 from matplotlib import pyplot as plt
-import joebvpfit
-import makevoigt
-import atomicdata
+from joebvp import makevoigt
+from joebvp import atomicdata
 from linetools.spectra.io import readspec
 from linetools.spectra.xspectrum1d import XSpectrum1D
+from linetools.lists.linelist import LineList
 from astropy.table import Table,vstack
 from astropy.io import ascii
 try:
-    import joebvp_cfg as cfg
+    from joebvp import joebvp_cfg as cfg
```

Comment:
This should be removed, because the idea was to import a local configuration file.

Reply:
Should be 
`try:
    import joebvp_cfg`
Right?

Reply:
Corrected